src/components/PaymentSection.jsx

```diff
@@ -15,7 +15,7 @@ function PaymentSection() {
       <p className="payment-para">
         Start with our free plan and switch to premium as you grow.{" "}
       </p>
-      <div className="card-container">
+      <div className="payment-card-container">
```

Comment:
why was this changed

Reply:
the class name was used twice in two different components causing issues with mobile styles in the `CardSection` component. So renamed class with a more explicit name `payment-card-container`

Reply:
changes approved